examples/looker-oauth/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "looker-oauth",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "develop": "webpack serve --host=0.0.0.0 --https --disable-host-check --config webpack.dev.config.js"
+  },
+  "dependencies": {
```

Comment:
My concern about the examples is the difficulty we have keeping the versions up to date. We have an issue with the access token server which I'm thinking should be moved to packages and eventually deleted when artifacts is done.

I don't know of a better place to put the example. I've been thinking of writing a script for examples that allows me to update versions easily. May be the script should go here and it can be run after each publish and eventually automated? Perhaps a script to ensure that examples have been updated to the current version when we do a publish. Maybe we can have a CI job update the packages post publish. 

Thinking out loud here. Totally unrelated to the merits of this PR.

Reply:
I think you've created this script now, @bryans99?